internal/uvm/create.go

```diff
@@ -119,13 +119,14 @@ func verifyOptions(ctx context.Context, options interface{}) error {
 // If `owner` is empty it will be set to the calling executables name.
 func newDefaultOptions(id, owner string) *Options {
 	opts := &Options{
-		ID:                    id,
-		Owner:                 owner,
-		MemorySizeInMB:        1024,
-		AllowOvercommit:       true,
-		EnableDeferredCommit:  false,
-		ProcessorCount:        defaultProcessorCount(),
-		FullyPhysicallyBacked: false,
+		ID:                      id,
+		Owner:                   owner,
+		MemorySizeInMB:          1024,
+		AllowOvercommit:         true,
+		EnableDeferredCommit:    false,
+		ProcessorCount:          defaultProcessorCount(),
+		ExternalGuestConnection: true,
```

Comment:
You should also remove the line in create_lcow.go that sets this to true, since it's a default across both LCOW and WCOW now.